packages/devtools_app/lib/src/utils.dart

```diff
@@ -966,7 +966,8 @@ extension LogicalKeySetExtension on LogicalKeySet {
     return sortedKeys.map((key) {
       if (_modifiers.contains(key)) {
         if (isMacOS && key == LogicalKeyboardKey.meta) {
-          return '⌘';
+          // TODO(issue #3352) Switch back to using ⌘ once supported on web.
+          return kIsWeb ? 'Command-' : '⌘';
```

Comment:
will there be a space on the other side of this before the hotkey? e.g. `Command- P`, or will it be `Command-P`? Either way, we should have a consistent number of spaces on either side of this dash. Also, slight preference for `cmd` to be more similar to how we write an abbreviated `ctrl`. If we use `cmd`, I don't think the dash is necessary.

Reply:
No space before the "P", it looks like this:

![image](https://user-images.githubusercontent.com/21270878/132917969-977743ea-864e-4637-b6ee-e5a5aa53adb8.png)

I chose `Command` to be consistent with how we are showing the modifier names here: https://github.com/flutter/devtools/blob/master/packages/devtools_app/lib/src/utils.dart#L943-L948, but happy to change to `cmd` instead. Wdyt?

Reply:
Oh, I thought we were using `ctrl`. In that case, lgtm on what you have!

Reply:
You may need to skip the 'meta mac' test for web. See other examples of this `skip: kIsWeb`. If you skip that test add a TODO link to stop skipping that links to the issue you filed